pkg/cloud/services/elb/loadbalancer_test.go

```diff
@@ -1000,6 +1003,109 @@ func TestChunkELBs(t *testing.T) {
 	}
 }
 
+func TestInstancePortCustomizationsInELB(t *testing.T) {
+	tests := []struct {
+		name   string
+		kcp    *controlplanev1.KubeadmControlPlane
+		mocks  func(m *mock_ec2iface.MockEC2APIMockRecorder)
+		expect func(t *testing.T, res *infrav1.ClassicELB)
+	}{
+		{
+			name: "load balancer config with kcp bind port customization",
+			kcp: &controlplanev1.KubeadmControlPlane{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test-cp",
+					Namespace: "foo",
+				},
+				TypeMeta: metav1.TypeMeta{
+					Kind:       "KubeadmControlPlane",
+					APIVersion: "kubeadm.k8s.io/v1beta2",
+				},
+				Spec: controlplanev1.KubeadmControlPlaneSpec{
+					KubeadmConfigSpec: cabpkv1.KubeadmConfigSpec{
+						InitConfiguration: &cabpkv1.InitConfiguration{
+							LocalAPIEndpoint: cabpkv1.APIEndpoint{
+								BindPort: 443,
+							},
+						},
+					},
+				},
+			},
+			mocks: func(m *mock_ec2iface.MockEC2APIMockRecorder) {},
+			expect: func(t *testing.T, res *infrav1.ClassicELB) {
+				if !res.Attributes.CrossZoneLoadBalancing {
+					t.Error("Expected load balancer to have cross-zone load balancing enabled")
```

Comment:
Do we need `CrossZoneLoadBalancing` check here?

Reply:
No, we don't, I will remove it.